pallets/confidential/src/lib.rs

```diff
@@ -79,6 +88,36 @@ decl_module! {
 
         fn deposit_event() = default;
 
+        #[weight = 400_000]
+        pub fn store_valid_asset_ids(origin, // TODO: how to ensure only privileged users can call this?
```

Comment:
We need a place on the chain to store the list of valid asset ids (encrypted form of ticker names). How can I enforce that only authorized entities can call and update that list.

Reply:
Sounds like you need something like:
```rust
T::CommitteeOrigin::ensure_origin(origin)?;
```
(see e.g., pips pallet)

Reply:
Thanks @Centril